packages/meditrak-app/app/widgets/Button.js

```diff
@@ -29,12 +42,17 @@ export function Button(props) {
       analyticsLabel={title}
       onPress={onPress}
     >
-      <Text style={[localStyles.text, textStyle]}>{title}</Text>
+      {Label}
```

Comment:
Seems like a change that could affect a bunch of things. Code wise it looks ok, but just wondering if it's best to do now?

Reply:
Good thinking. I did a bit of a snoop around during development and I think it's safe. I figure that meditrak will be getting a bit of testing over the next few days so I reckon the QAs will catch any bugs. Happy to see how it goes and fix/rollback if there are issues